CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.48.0      2023-01-26
+
+* Update to Go 1.19
+* Bugfix: Ensure relfrozenxid = 0 is tracked as full frozenxid = 0 (instead of
+  adding epoch prefix)
```

Comment:
Additional changelog entries for the PRs I just merged:

```
* Amazon RDS and Amazon Aurora: Support IAM token authentication
  - This adds a new configuration setting, db_use_iam_auth / DB_USE_IAM_AUTH
     that, if enabled, causes the collector to fetch a short-lived token for
     logging into the database instance from the AWS API, instead of using a
     hardcoded password in the collector configuration file.
  - In order to use this setting IAM authentication needs to be enabled on the
    database instance / cluster, and the pganalyze IAM policy needs to be
    extended to cover the "rds-db:connect" privilege for the pganalyze user:
    https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html
* Amazon RDS: Avoid DescribeDBInstances calls for log downloads for RDS instances
  - This should reduce issues with rate limiting on this API call in some cases
* Log parsing: Add support for timezones specified by number, such as "-03"
```

(I think we can omit the change that removed `rockylinux9` from the package tests, since thats not user visible)